src/main/java/be/seeseemelk/mockbukkit/entity/ChickenMock.java

```diff
@@ -46,30 +51,25 @@ public boolean isBreedItem(@NotNull ItemStack stack)
 	@Override
 	public boolean isChickenJockey()
 	{
-		//TODO: Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.isChickenJockey;
 	}
 
 	@Override
 	public void setIsChickenJockey(boolean isChickenJockey)
 	{
-		//TODO: Auto-generated method stub
-		throw new UnimplementedOperationException();
-
+		this.isChickenJockey = isChickenJockey;
```

Comment:
How does this work in Paper? Should we be spawning the baby zombie and setting it to ride the chicken?

Reply:
From what I can tell it's just a variable thats set. I don't see the spawning of the baby zombie anywhere

Reply:
Yep, it's just a boolean that is set to indicate things for the chicken, for example not to lay eggs, the xp rewards, and whether to despawn when player is too far away.